Update config-settings.rst (#3378)
mattermost/docs: source/administration/config-settings.rst

| This feature's ``config.json`` setting is ``"IdpDescriptorUrl": ""`` with string input.                                                                              |
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+

Identity Provider Metadata URL
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> The metadata URL for the Identity Provider you use for SAML requests.

Reviewer: ```suggestion
The URL where Mattermost sends a request to obtain setup metadata from the provider.
```

| This feature's ``config.json`` setting is ``"Enable": true`` with options ``true`` and ``false``.                                                                    |
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+

Automatic Prepackaged Plugins
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> **True**: The server will detect previously enabled plugins on the server and will automatically install them.

Reviewer 2: ```suggestion
**True**: Any pre-packaged plugins enabled in the configuration will be installed or upgraded automatically. If a newer version is already installed, no changes are made.
```

> **False**: The server will not detect previously enabled plugins on the server.

Reviewer 2: ```suggestion
**False**: Pre-packaged plugins are not installed or upgraded automatically but may be installed manually from the Plugin Marketplace, even when offline.
```

| This feature's ``config.json`` setting is ``"EnableMarketplace": true`` with options ``true`` and ``false``.                                                         |
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+

Enable Remote Marketplace
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> **True**: Marketplace fetches latest plugins from the configured Marketplace URL.

Reviewer 2: ```suggestion
**True**: The server will attempt to connect to the configured Plugin Marketplace to show the latest plugins. If the connection fails, the Plugin Marketplace shows only pre-packaged and already installed plugins alongside a connection error.
```

> **False**: Marketplace does not fetch latest plugins from the configured Marketplace URL.

Reviewer 2: ```suggestion
**False**: The server will not attempt to connect to a remote marketplace, instead showing only pre-packaged and already installed plugins. Use this setting if your server cannot connect to the internet.
```
